docs/features/multiple-networks/multi-homing.md

```diff
@@ -1,33 +1,96 @@
 # Multihoming
-A K8s pod with more than one network interface is said to be multi-homed. The
-[Network Plumbing Working Group](https://github.com/k8snetworkplumbingwg/multi-net-spec)
-has put forward a [standard](https://github.com/k8snetworkplumbingwg/multi-net-spec)
-describing how to specify the configurations for additional network interfaces.
 
-There are several delegating plugins or meta-plugins (Multus, Genie)
-implementing this standard.
+## Introduction
 
-After a pod is scheduled on a particular Kubernetes node, kubelet will invoke
-the delegating plugin to prepare the pod for networking. This meta-plugin will
-then invoke the CNI responsible for setting up the pod's default cluster
-network, and afterwards it iterates the list of additional attachments on the
-pod, invoking the corresponding delegate CNI implementing the logic to attach
-the pod to that particular network.
+Multihoming allows the configuration of secondary-network interfaces for K8s pods.
+OVN-Kubernetes secondary-network has three configurable topologies: layer 3, layer 2, or localnet.
+
+- A layer 3 topology is a simplification of the topology for the cluster default network - but without egress.
+- A layer 2 topology provides an isolated (no egress) cluster-wide layer2 network providing east-west traffic to the
+  cluster workloads.
+- A localnet topology is based on layer 2 topology, but also allows connecting to an existent (configured) physical
+  network to provide north-south traffic to the workloads.
+
+For layer 2 and localnet topologies, multihoming also allows IP features on secondary interfaces such as static IP
+allocation and persistent IP addresses for virtualization workloads.
 
-## Configuring secondary networks
 To allow pods to have multiple network interfaces, the user must provide the
 configurations specifying how to connect to these networks; these
-configurations are defined in a CRD named `NetworkAttachmentDefinition`.
+configurations are set in a CRD named `NetworkAttachmentDefinition`, defined by
+the [Kubernetes Network Custom Resource Definition De-facto Standard](https://github.com/k8snetworkplumbingwg/multi-net-spec).
+
+Notes:
+* layer 2 and layer 3 topologies are overlays - thus, they do not need **any** previous physical network configuration.
+
+## Prerequisites
+
+- multus-cni
```

Comment:
please turn this into a link to the upstream project.

Reply:
DONE